### ISSUE TITLE
Improve led-matrix to display the image on the box

### ISSUE THREAD

**Issue by Natim** [magopian/elm-fun#1]
- [ ] Allow to copy the code of the matrix as an hex string (i.e: `00000000000000000040200001801800000000000030c0000030c00000000000000600000006000000c030000060300000602000003fe0000000000000000000`)
- [ ] Load the list of pages from https://kinto.dev.mozaws.net/v1/buckets/default/collections/laboite/records using a login/password default: laboite:laboite)
- [ ] Allow to preview pages
- [ ] Allow to update pages
- [ ] Allow to create pages
- [ ] Allow to change duration of the page
- [ ] Allow to change intensity of the page

**Comment by Natim** [comment]
![img_20171003_110413](https://user-images.githubusercontent.com/229453/31117649-ea7e8bb8-a82a-11e7-9071-1338f70e9a00.jpg)

**Comment by magopian** [comment]
For the first checkbox: what's the format of the hex string? Is it one byte per 8x8 led matrix line (with 8 led matrices iirc)? That would need 2 characters per line, so 16 per matrix, so 128 for the whole display.

For all the other checkboxes, I'm not sure how that would fit in the scope of the "led matrix" small elm project. I guess the other checkboxes would make sense to offer a possibility to upload/load a design to kinto?

**Comment by Natim** [comment]
>  I'm not sure how that would fit in the scope of the "led matrix" small elm project

Yes we can probably fork it.

**Comment by Natim** [comment]
> For the first checkbox: what's the format of the hex string? I first thought it was one byte per 8x8 led matrix line (with 8 led matrices iirc), but that would need only 64 characters, and it seems there's the double (128).

Yes I agree that with a string we could probably divide the lenght by 2.

**Comment by magopian** [comment]
Your answer above might be wrong, I edited my original post, it seems it's the correct number of characters after all ;)

**Comment by Natim** [comment]
Maybe we could also use a base64 string instead (6bits per characters rather than 4 with the hex string 86 rather than 128)

ping @jlebunetel
